Formula/libjwt.rb

```diff
@@ -0,0 +1,22 @@
+class Libjwt < Formula
+  desc "JWT C Library"
+  homepage "https://benmcollins.github.io/libjwt/"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "openssl" => :build
+  depends_on "check" => :build
+
+  def install
+    system "autoreconf", "-i"
```

Comment:
This will need `autoconf`, `automake` and `libtool` to be present for `:build`.

Reply:
Indeed! Checking my own PR for somebody else's comments would have saved me one hour of figuring this myself 🤦‍♂️ 

Thank you very much anyway 🙂